src/generator/common/helpers.ts

```diff
@@ -5,41 +5,56 @@
 
 import { Session } from '@azure-tools/autorest-extension-base';
 import { comment } from '@azure-tools/codegen'
-import { CodeModel, Language } from '@azure-tools/codemodel';
-import { values } from '@azure-tools/linq';
+import { CodeModel, ImplementationLocation, Language, Operation } from '@azure-tools/codemodel';
+import { length, values } from '@azure-tools/linq';
+
+type importEntry = { imp: string, alias?: string };
 
 // tracks packages that need to be imported
 export class ImportManager {
-  private imports: string[];
+  private imports: importEntry[];
 
   constructor() {
-    this.imports = new Array<string>();
+    this.imports = new Array<importEntry>();
   }
 
   // adds a package for importing if not already in the list
-  add(imp: string) {
+  // accepts an optional package alias.
+  add(imp: string, alias?: string) {
     for (const existing of values(this.imports)) {
-      if (existing === imp) {
+      if (existing.imp === imp) {
         return;
       }
     }
-    this.imports.push(imp);
+    this.imports.push({ imp: imp, alias: alias });
   }
 
   // returns the number of packages in the list
   length(): number {
     return this.imports.length
   }
 
+  // returns the import list as Go source code
   text(): string {
-    this.imports.sort(SortAscending);
+    if (this.imports.length === 1) {
+      const first = this.imports[0];
+      return `import ${this.alias(first)}"${first.imp}"\n\n`;
```

Comment:
is this for the azinternal package?

Reply:
Yeah needed to support package aliases for it.